frontend/src/config/rbd/translation.json

```diff
@@ -103,5 +103,29 @@
   "Normalized Difference Vegetation Index (NDVI) anomaly when compared to the long-term average": "Anomalie de l'indice de végétation par différence normalisée (NDVI) par rapport à la moyenne à long terme",
   "Daytime land surface temperature": "Température diurne au sol",
   "Daytime land surface temperature anomaly when compared to long-term average": "Anomalie de la température diurne au sol par rapport à la moyenne à long terme",
+  "Land Surface Temperature - 10-day Amplitude (MODIS)": "Température de surface - 10 Jours d'Amplitude (MODIS)",
+  "Land Surface Temperature Amplitude refers to the difference between the maximum and minimum temperature in degrees Celsius": "L'amplitude de la température au sol en référence à la différence entre la température maximale et minimale en degrés Celsius",
+  "Nighttime Land Surface Temperature - 10-day (MODIS)": "Température nocturne au sol - 10 jours (MODIS)",
+  "Nighttime land surface temperature": "Température nocturne au Sol",
+  "Integrated Context Analysis (ICA)": "Analyse Intégrée du Contexte (AIC)",
+  "Integrated Context Analysis": "Analyse Intégrée du Contexte",
+  "Food insecurity recurrence": "Récurrence de l'Insécurité Alimentaire",
+  "Drought recurrence": "Récurrence de la Sécheresse",
+  "Flood recurrence": "Récurrence des inondations",
+  "Landslide recurrence": "Récurrence des Glissements de Terrain",
+  "Combined hazard risk": "Risque Combiné de Chocs Naturels",
+  "ICA area classification": "Classification de Zone AIC",
+  "Risk Analysis": "Analyse de Risque",
+  "Multidimensional Risk Analysis": "Analyse de Risque Multidimensionel",
```

Comment:
Is it the analysis or the risk that are multidimensional?

If it's the analysis, it should be `Multidimensionelle`

Reply:
thanks. I wouldn't focus too much on the translations. I'll get final input from the team in Dakar on this in the next week or two.